data_diff/sqeleton/databases/base.py

```diff
@@ -59,10 +59,10 @@ def _inner():
             try:
                 return f()
             except ModuleNotFoundError as e:
-                s = text
                 if package:
-                    s += f"You can install it using 'pip install data_diff[{package}]'."
-                raise ModuleNotFoundError(f"{e}\n\n{s}\n")
+                    print(f"Missing database dependency, complete setup by running 'pip install data_diff[{package}]'.\n")
+                    exit()
+                raise ModuleNotFoundError(f"{e}\n")
```

Comment:
Since the extra hint is gone and it is now the same exception of the same type with the same message, we should rather do `raise` (no arguments) in the end and let the original exception with the original stack trace escalate.

Reply:
The issue I see is around presenting an expected user workflow as an error

Reply:
I reverted to the original raise -- showing the word "error" is pretty minor